cmd/minikube/cmd/start.go

```diff
@@ -153,5 +155,6 @@ func init() {
 	startCmd.Flags().StringVarP(&vmDriver, "vm-driver", "", constants.DefaultVMDriver, fmt.Sprintf("VM driver is one of: %v", constants.SupportedVMDrivers))
 	startCmd.Flags().IntVarP(&memory, "memory", "", constants.DefaultMemory, "Amount of RAM allocated to the minikube VM")
 	startCmd.Flags().IntVarP(&cpus, "cpus", "", constants.DefaultCPUS, "Number of CPUs allocated to the minikube VM")
+	startCmd.Flags().IntVar(&disk, "disk-size", constants.DefaultDiskSize, "Disk size in gigabytes allocated to the minikube VM")
```

Comment:
Wonder if this should be a flag that takes units, e.g. 20G, rather than just an int?